CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ##### Breaking
 
-* None.
+* `--swift-version` now defaults to 2.1.1 instead of 2.1.
+  [Nikita Lutsenko](https://github.com/nlutsenko)
+  [#416](https://github.com/realm/jazzy/pull/416)
 
 ##### Enhancements
 
 * Add `--sdk [iphone|watch|appletv][os|simulator]|macosx` option for Objective-C
-  projects.  
```

Comment:
All this whitespace was intentional. Markdown treats it as a line break in the rendered output.